pages/common/npx.md

```diff
@@ -0,0 +1,15 @@
+# npx
+
+> Execute binaries from npm packages.
+
+- Execute the command:
+
+`npx <command>[@version] [command-arg]`
```

Comment:
This is copied verbatim from the official description. We do not use syntax like <>. Please have a look at the CONTRIBUTING guidelines for the token syntax. 

And start with a simple example like `npm {{command}}` and then move to complex examples.

Reply:
sorry, I will try again